src/documents/articles/what-kind-of-web-components-do-we-need.html.md

```diff
@@ -0,0 +1,14 @@
+---
+title: What kind of Web Components do we need?
+authors: [hidde_vries]
+date: 2015-07-06
+original_date: 2015-07-02
+link: https://hiddedevries.nl/en/blog/2015-07-02-what-kind-of-web-components-do-we-need
+category: articles
+layout: single
+tags: ['Web Components', 'Custom elements']
```

Comment:
Everything here is about Web Components, doesn't make sense to include such tag.